src/common/display.c

```diff
@@ -178,6 +180,9 @@ void guac_common_display_dup(guac_common_display* display, guac_user* user,
     guac_common_display_dup_layers(display->layers, user, socket);
     guac_common_display_dup_layers(display->buffers, user, socket);
 
+    /* Synchronize all connected users */
```

Comment:
This particular line does not synchronize all connected users, but rather marks the boundary of the first frame.